packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -115,17 +127,16 @@ const useRootStyles = makeStyles({
   },
   'filled-warning': {
     backgroundColor: tokens.colorPaletteYellowBackground3,
-    color: tokens.colorNeutralForeground1,
-    ...shorthands.borderColor(tokens.colorPaletteYellowBackground3),
+    color: tokens.colorNeutralForeground1Static,
```

Comment:
The text isn't visible in high contrast ([link to Screener](https://screener.io/v2/states/4V5IW.microsoft-fluentui-react-components/badge-font-styles/1024x768/Chrome/badge-converged-tint-high-contrast-0))

<img width="268" alt="tint_high_contrast" src="https://user-images.githubusercontent.com/93940821/172736017-d8f89a8c-1132-4933-8d56-e8c93c979700.png">